utils/log.py

```diff
@@ -470,54 +470,60 @@ def slaveid(self):
         from fixtures.artifactor_plugin import SLAVEID
         return SLAVEID or ""
 
-    def art_log(self, level_name, message, kwargs):
+    def art_log(self, level_name, message, args, kwargs):
+        try:
+            formatted_message = message % args
+        except TypeError:
+            formatted_message = message
+            self.logger.error('Could not format the string %r with %r', message, args)
```

Comment:
Empty args is valid. This is a defensive line, protecting from accidental wrong argument count.